Sources/Prometheus/Docs.docc/swift-metrics.md

```diff
@@ -111,28 +111,19 @@ generated in a third party library.
 
 #### Default buckets
 
-Swift Metric ``Timer``s are backed by a Prometheus ``DurationHistogram`` and Swift Metric 
-``Recorder``s that aggregate are backed by a Prometheus ``ValueHistogram``. As a user, you can 
+Swift Metric ``Timer``s are backed by a Prometheus ``Histogram`` and Swift Metric 
+``Recorder``s that aggregate are also backed by a Prometheus ``Histogram``. As a user, you can 
 specify which buckets shall be used within the backing ``Histogram``s.
 
 ```swift
 var factory = PrometheusMetricsFactory()
 
-factory.defaultDurationHistogramBuckets = [
-  .milliseconds(5),
-  .milliseconds(10),
-  .milliseconds(25),
-  .milliseconds(50),
-  .milliseconds(100),
+factory.defaultTimerHistogramBuckets = [
+  0.005, 0.01, 0.025, 0.05, 0.1
 ]
```

Comment:
This feels like a step backwards as well tbh

Reply:
I agree that this feels like a step backwards